css-ecommerce-page/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="row">
+    <p class="header font-style">New Arrivals</p>
+  </div>
+  <div class="row">
+    <div class="column">
+      <img class="object-contain" src="images/dress1.jpeg" alt="dress1">
+    </div>
+    <div class="column">
+      <img class="object-contain" src="images/ysl-bag.jpeg" alt="ysl-bag">
+    </div>
+    <div class="column">
+      <img src="images/key.jpeg" alt="key">
+    </div>
+    <div class="column">
+      <img src="images/lv-bag.jpg" alt="lv-bag">
+    </div>
+    <div class="column">
+      <img src="images/dress2.jpeg" alt="dress2">
+    </div>
+  </div>
+  <div class="row font-style text-direction">
+    <div class="column">
+      <p class="brand">Free People<span class="size"> | Size 6</span></p>
+      <span class="new-price">$35</span>
+      <span class="old-price"> $25 retail</span>
+    </div>
+    <div class="column">
+      <p class="brand">Yves Saint Laurent <span class="size"> | One-Size</span></p>
+      <span class="new-price">$200</span>
+      <span class="old-price"> $600 retail</span>
+    </div>
+    <div class="column">
+      <p class="brand">Tiffany<span class="size"> | One-Size</span></p>
+      <span class="new-price">$476</span>
+      <span class="old-price"> $2000 retail</span>
+    </div>
+    <div class="column">
+      <p class="brand">Louis Vuitton<span class="size"> | One-Size</span></p>
+      <span class="new-price">$95</span>
+      <span class="old-price"> $1200 retail</span>
+    </div>
+    <div class="column">
+      <p class="brand">Reformation <span class="size"> | Size 4</span></p>
+      <span class="new-price">$95</span>
+      <span class="old-price"> $198 retail</span>
+    </div>
+  </div>
```

Comment:
You should have the images and the text for the images in their own `column` class element together, not in separate `row` elements. 

Also, please give your `column` class a name that describes its size.